bundle/README.md

```diff
@@ -152,3 +152,111 @@ To configure `bookinfo` with a gateway using `Gateway API`:
     $ echo "http://${GATEWAY_URL}/productpage"
     ```
 
+## Istio Addons Integrations
+
+Istio can be integrated with other software to provide additional functionality (More information can be found in: https://istio.io/latest/docs/ops/integrations/). The Sail Operator provides support for the following integrations:
+
+### Prometheus
+
+`Prometheus` is an open-source systems monitoring and alerting toolkit. You can use `Prometheus` with the Sail Operator to keep an eye on how healthy Istio and the apps in the service mesh are, for more information, see [Prometheus](https://istio.io/latest/docs/ops/integrations/prometheus/). 
+
+To install Prometheus you will need to follow this steps:
+
+1. Deploy the operator:
```

Comment:
As this isn't an operator, perhaps just say "Deploy Prometheus" (Same comment for the other sections with "Deploy the operator")